cloud/google/machineactuator.go

```diff
@@ -709,6 +699,27 @@ func (gce *GCEClient) getImagePath(img string) (imagePath string) {
 	return defaultImg
 }
 
+func buildDisks(config *gceconfig.GCEProviderConfig, zone string, imagePath string, minDiskSizeGb int64) []*compute.AttachedDisk {
+	var disks []*compute.AttachedDisk
+	for idx, disk := range config.Disks {
+		diskSizeGb := disk.InitializeParams.DiskSizeGb
+		if diskSizeGb < minDiskSizeGb {
+			diskSizeGb = minDiskSizeGb
```

Comment:
Might want to spit out a warning log that the configuration is bad and you are defaulting to the min.

Reply:
Yes, will do.